app/services/jobseekers/job_applications/quick_apply.rb

```diff
@@ -28,7 +26,10 @@ def recent_job_application
   end
 
   def attributes_to_copy
-    personal_details_fields + professional_status_fields + employment_history_fields + ask_for_support_fields
+    ::Jobseekers::JobApplication::PersonalDetailsForm.fields +
```

Comment:
What does the initial `::` do?

Reply:
I've removed them now, good shout. I briefly had this class definition wrapped into a module and that confused the constant lookup.

i.e. it was

Reply:
```ruby
module Jobseekers::JobApplications
  class QuickApply
    # ...

Reply:
But basically `::` at the beginning says I explicitly intend this to be at the root of the namespace hierarchy, not nested within whatever module I'm in